util/libcrypto.num

```diff
@@ -5347,3 +5346,8 @@ EVP_ASYM_CIPHER_description             ?	3_0_0	EXIST::FUNCTION:
 EVP_KEM_description                     ?	3_0_0	EXIST::FUNCTION:
 EVP_KEYEXCH_description                 ?	3_0_0	EXIST::FUNCTION:
 EVP_KDF_description                     ?	3_0_0	EXIST::FUNCTION:
+OSSL_PARAM_dup                          ?	3_0_0	EXIST::FUNCTION:
+OSSL_PARAM_merge                        ?	3_0_0	EXIST::FUNCTION:
+OSSL_PARAM_free                         ?	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_todata                         ?	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_export                         ?	3_0_0	EXIST::FUNCTION:
```

Comment:
There isn't an explicit test of `EVP_PKEY_export`. Do we need one, or is it sufficiently tested via `EVP_PKEY_todata`?

Reply:
Added an export test..